docs/how-to/account-info.md

```diff
@@ -2,13 +2,13 @@
 
 When you connect your PMM instances to Percona Platform, PMM gets access to:
 
--  more alert templates 
+-  More alert templates
 -  Basic Advisor Checks for additional database checks
 -  Standard/Premium Advisor Checks for more advanced database health checks.
-  
+
 Standard/Premium checks are available when you connect to Percona Platform with a customer account.
 
-You can check the list of available Standard/Premium Advisor checks in the [Percona Platform documentation](https://docs.percona.com/percona-platform/checks.html) or on the [Percona Platform Subscription plans page](https://www.percona.com/software/percona-platform/subscription).
+You can check the list of available Standard/Premium Advisor checks in the [Percona Platform documentation](https://docs.percona.com/percona-platform/checks.html).
```

Comment:
we can now use https://docs.percona.com/percona-monitoring-and-management/get-started/advisors-details.html as we moved (copied , for now but planing to remove from portal) the list of Advisors in PMM doc

Reply:
okay thanks. Indeed this page you mentioned is quite complete, even more than the one on the portal documentation (https://docs.percona.com/percona-platform/advisors.html) which has the list of advisors, but not the list of checks for each advisors. Implementing the change